osf/models/user.py

```diff
@@ -102,6 +102,14 @@ def create_superuser(self, username, password):
         return user
 
 
+class Email(BaseModel, TimeStampedModel):
+    address = LowercaseEmailField(unique=True, db_index=True, validators=[validate_email])
+    user = models.ForeignKey('OSFUser', related_name='emails', null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
Currently, it should not. The PR this was originally inspired by had email objects associated with models other than just`OSFUser`. I can remove nullability until that is true again